wdl-lint/src/rules/container_uri.rs

```diff
@@ -93,13 +93,13 @@ fn array_containing_anys(spans: impl Iterator<Item = Span>) -> Diagnostic {
     diagnostic
 }
 
-impl Rule for ContainerValue {
+impl Rule for ContainerUri {
     fn id(&self) -> &'static str {
         ID
     }
 
     fn description(&self) -> &'static str {
-        "Ensures that values for the `container` key within `runtime`/`requirements` sections are \
+        "Ensures that the value for the `container` key in `runtime`/`requirements` sections is \
```

Comment:
I think this should be reverted. The container key supports arrays (and I think this rule handles those), so it can be plural.